packages/core/overlays/core-overlays-browser-internal/BUILD.bazel

```diff
@@ -0,0 +1,124 @@
+load("@npm//@bazel/typescript:index.bzl", "ts_config")
+load("@build_bazel_rules_nodejs//:index.bzl", "js_library")
+load("//src/dev/bazel:index.bzl", "jsts_transpiler", "pkg_npm", "pkg_npm_types", "ts_project")
+
+PKG_DIRNAME = "core-overlays-browser-internal"
+PKG_REQUIRE_NAME = "@kbn/core-overlays-browser-internal"
+
+SOURCE_FILES = glob(
+  [
+    "src/**/*.ts",
+    "src/**/*.tsx",
+    "src/**/*.scss"
```

Comment:
package contains `.scss` files

Reply:
FWIW, a11y frt tests for discover had an error on removing them from the package.